src/app/screens/Calculator/utils.js

```diff
@@ -0,0 +1,3 @@
+const changeScreenToRecord = history => history.push('/record');
+
+export default changeScreenToRecord;
```

Comment:
If you aren't using this, please delete this file. 
Anyway, when we create a new constants/function/whatever export it directly, theres no need of doing a separated `export default`
You can do `export const yourConstName = () => ...`